src/options/optionsMain.js

```diff
@@ -154,8 +154,8 @@ function optionsFactory(inpOptions, callback) {
 
 function sanitizeFilename(str) {
     const cleaned = sanitizeFilenameLib(str);
-    // remove non ascii
-    return cleaned.replace(/[^\x00-\x7F]/, '');
+    // remove all non ascii or do nothing
+    return cleaned.replace(/[^\x00-\x7F]/g, '') || cleaned;
```

Comment:
what if the filename is all none ascii characters, there needs a default name string.